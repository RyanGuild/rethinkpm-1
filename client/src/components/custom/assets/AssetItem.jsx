import React from 'react';
import PropTypes from 'prop-types';

const AssetItem = (props) => {
  return (
    <div>
      <h2 className="my-2">My Properties</h2>
      <table className="table">
        <thead>
          <tr>
            <th>Company</th>
            <th className="hide-sm">Title</th>
            <th className="hide-sm">Years</th>
            <th />
          </tr>
        </thead>
        <tbody></tbody>
      </table>
      <p>ooh ya</p>
    </div>
  );
};

AssetItem.propTypes = {};

export default AssetItem;
