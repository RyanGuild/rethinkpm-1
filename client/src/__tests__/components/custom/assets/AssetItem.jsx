import React from 'react';
import {render} from '@testing-library/react';
import AssetItem from '../../../../components/custom/assets/AssetItem';

test('render and snapshot', () => {
  let {container} = render(<AssetItem />);
  expect(container).toMatchSnapshot();
});
