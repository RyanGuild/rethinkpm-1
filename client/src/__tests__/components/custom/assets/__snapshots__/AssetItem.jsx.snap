// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render and snapshot 1`] = `
<div>
  <div>
    <h2
      class="my-2"
    >
      My Properties
    </h2>
    <table
      class="table"
    >
      <thead>
        <tr>
          <th>
            Company
          </th>
          <th
            class="hide-sm"
          >
            Title
          </th>
          <th
            class="hide-sm"
          >
            Years
          </th>
          <th />
        </tr>
      </thead>
      <tbody />
    </table>
    <p>
      ooh ya
    </p>
  </div>
</div>
`;
